checker/src/main/java/org/checkerframework/checker/optional/RevisedOptionalChecker.java

```diff
@@ -1,14 +1,16 @@
 package org.checkerframework.checker.optional;
 
-import java.util.Set;
+import java.util.ArrayList;
+import java.util.Collection;
 import org.checkerframework.checker.nonempty.NonEmptyChecker;
-import org.checkerframework.common.basetype.BaseTypeChecker;
+import org.checkerframework.framework.source.AggregateChecker;
+import org.checkerframework.framework.source.SourceChecker;
 
-public class RevisedOptionalChecker extends BaseTypeChecker {
+public class RevisedOptionalChecker extends AggregateChecker {
```

Comment:
Without this change, the RevisedOptionalChecker was a new checker that re-used the `@Present` qualifiers but with the standard BaseTypeFactory and BaseTypeChecker.  Which isn't what we want right?  Basically we want to just rename the `NonEmptyChecker` to the `RevisedOptionalChecker`, right?

(I was running the NonEmptyChecker directly, so I didn't notice this problem before.)
javacheck -processor RevisedOptionalChecker OptionalNonEmptyTest.java now passes.